components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2638,6 +2647,10 @@ void RewardsServiceImpl::StartAutoContributeForTest() {
   bat_ledger_->StartAutoContribute();
 }
 
+void RewardsServiceImpl::CheckInsufficientFundsForTesting() {
```

Comment:
See above

Reply:
These are call ins so we don't have to wait for a timer.